src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -57,13 +59,37 @@ public void setShowTime(LocalDateTime showTime) {
         this.showTime = showTime;
     }
 
+    public MovieSession setShowTime(int year, int month, int day, int hour, int minute) {
+        this.showTime = showTime.of(year, month, day, hour, minute);
```

Comment:
The `setShowTime` method incorrectly uses `showTime.of(...)`. It should be `LocalDateTime.of(...)` to create a new `LocalDateTime` instance.